tasklist/client/e2e/pages/FormJSDetailsPage.ts

```diff
@@ -50,13 +51,33 @@ class FormJSDetailsPage {
     this.selectDropdown = this.form.getByText('Select').last();
     this.checkbox = this.form.getByLabel('Checkbox');
     this.tagList = page.getByPlaceholder('Search');
+    this.dynamicListAddnewBtn = page.getByRole('button', {name: /add new/i});
```

Comment:
This is just a generic button, we can simply inline `page.getByRole('button', {name: /add new/i})` this we won't have create a new element on the POM everytime we use a new form